<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>beaglemain</class>
 <widget class="QMainWindow" name="beaglemain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>beaglemain</string>
  </property>
  <widget class="QWidget" name="main">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,6,0,2" columnstretch="25,1,0,9">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="header_left_layout">
        <item alignment="Qt::AlignHCenter">
         <widget class="QComboBox" name="Mode_Change">
          <item>
           <property name="text">
            <string>Audio</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Video</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="4">
       <layout class="QVBoxLayout" name="mid_spacer">
        <item>
         <spacer name="midspacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="0" column="2" colspan="2">
       <layout class="QHBoxLayout" name="header_right_layout"/>
      </item>
      <item row="1" column="0" rowspan="3">
       <layout class="QHBoxLayout" name="browse_layout">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item row="1" column="3">
       <layout class="QVBoxLayout" name="playlist_layout">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item row="2" column="3">
       <layout class="QVBoxLayout" name="right_spacer">
        <item>
         <spacer name="rtspacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="2" colspan="2">
       <layout class="QVBoxLayout" name="control_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionImport_Video"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionImport">
   <property name="text">
    <string>Import Audio</string>
   </property>
  </action>
  <action name="actionImport_Video">
   <property name="text">
    <string>Import Video</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
